patches/tModLoader/Terraria/ModLoader/UI/UIModSources.cs

```diff
@@ -105,14 +105,14 @@ internal class UIModSources : UIState, IHaveBackButtonCommand
 				Top = { Pixels = -65 }
 			};
 			buttonBA.WithFadedMouseOver();
-			buttonBA.OnClick += BuildMods;
+			//buttonBA.OnClick += BuildMods;
```

Comment:
So the build all functionality is completely gone?

Reply:
It's been hidden from the UI for a while, this just removes more parts of it